articles/security/azure-operational-security-best-practices.md

```diff
@@ -53,38 +53,38 @@ Customers build an end-to-end network in Azure by combining network resources li
+**Detail**: [Infrastructure as code](https://docs.microsoft.com/en-us/azure/devops/learn/what-is-infrastructure-as-code) is a set of techniques and practices that help IT pros remove the burden of day-to-day build and management of modular infrastructure. It enables IT pros to build and maintain their modern server environment in a way that’s like how software developers build and maintain application code.
 
 You can use [Azure Resource Manager](https://azure.microsoft.com/documentation/articles/resource-group-authoring-templates/) to provision your applications by using a declarative template. In a single template, you can deploy multiple services along with their dependencies. You use the same template to repeatedly deploy your application in every stage of the application lifecycle.
 
-**Best practice**: Automatically build and deploy to Azure web apps or cloud services.   
-**Detail**: You can configure your Visual Studio Team Services (VSTS) team projects to [automatically build and deploy](https://www.visualstudio.com/docs/build/overview) to Azure web apps or cloud services. VSTS automatically deploys the binaries after doing a build to Azure after every code check-in. The package build process is equivalent to the Package command in Visual Studio, and the publishing steps are equivalent to the Publish command in Visual Studio.
+**Best practice**: Automatically build and deploy to Azure web apps or cloud services.  
+**Detail**: You can use Azure Pipelines to [automatically build and deploy](https://docs.microsoft.com/en-us/azure/devops/pipelines/index?view=vsts) to Azure web apps or cloud services. Azure Pipelines automatically deploys the binaries after doing a build to Azure after every code check-in. The package build process is equivalent to the Package command in Visual Studio, and the publishing steps are equivalent to the Publish command in Visual Studio.
 
-**Best practice**: Automate release management.   
-**Detail**: Visual Studio [Release Management](https://msdn.microsoft.com/library/vs/alm/release/overview) is a solution for automating multiple-stage deployment and managing the release process. Create managed continuous deployment pipelines to release quickly, easily, and often. With Release Management, you can automate your release process, and you can have predefined approval workflows. Deploy on-premises and to the cloud, extend, and customize as required.
+**Best practice**: Use continuous deployment.  
+**Detail**: [Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/index?view=vsts) is a solution for automating multiple-stage deployment and managing the release process. Create managed continuous deployment pipelines to release quickly, easily, and often. With Release Management, you can automate your release process, and you can have predefined approval workflows. Deploy on-premises and to the cloud, extend, and customize as required.
 
```

Comment:
I missed a mention of Release Management here, that should be removed. "With Azure Pipelines..."